Sources/_StringProcessing/Regex/Match.swift

```diff
@@ -180,20 +180,13 @@ extension BidirectionalCollection where SubSequence == Substring {
 }
 
 @available(SwiftStdlib 5.7, *)
-extension Regex {
```

Comment:
Moving this away from Match.swift as `Regex` initializers are declared in Core.swift.